kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaproxies.kroxylicious.io-v1.yml

```diff
@@ -99,6 +99,9 @@ spec:
                       type:
                         description: type of condition in CamelCase or in foo.example.com/CamelCase.
                         type: string
+                    required:
```

Comment:
non-blocking. should we also be making `type`,`status`,`reason` and `message` required, they look to be required in the [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1623-standardize-conditions#kep-1623-standardize-condition)

Reply:
I was unaware of the KEP, it is worth calling out in the description of the PR that this PR to stepping toward KEP-1623.  We should probably raise an issue for what remains.

Reply:
Ah rats, I lost the commit message where I referenced it! Yes, the PR or commit should probably link to that.

Reply:
`status` is optional in KEP 1623 at least.